--- Makefile.orig	2022-08-17 00:05:27.000000000 +0300
+++ Makefile	2022-08-17 00:18:56.000000000 +0300
@@ -48,7 +48,7 @@
 
 FFTWDIR = $(TINKERDIR)/fftw
 FFTW_LIBDIR = -L$(FFTWDIR)/lib
-FFTW_LIBS = -lfftw3_threads -lfftw3
+FFTW_LIBS =  -L${MKLROOT}/lib/intel64 -liomp5 -lpthread -lm -ldl -lmkl_rt
 
 APBSDIR = $(TINKERDIR)/apbs
 APBS_INCDIR = -I$(APBSDIR)/include
@@ -84,16 +84,16 @@
 ##  Compiler:  GNU gfortran 8.1
 ##  Parallel:  OpenMP
 
-F77 = gfortran
-F77FLAGS = -c
+F77 = ifort
+F77FLAGS = -qmkl=parallel -c
 #OPTFLAGS = -Ofast -march=native -fopenmp
-OPTFLAGS = -Ofast -mssse3 -fopenmp
+OPTFLAGS = -march=core-avx2 -mfma -no-ipo -no-prec-div -recursive -qopenmp
 #OPTFLAGS = -Og -fbacktrace -fcheck=bounds -Wunused -Wmaybe-uninitialized
 LIBDIR = -L. -L$(TINKER_LIBDIR)/macos
 LIBS = 
 LIBFLAGS = -crusv
 RANLIB = ranlib -c
-LINKFLAGS = $(OPTFLAGS) -static-libgcc
+LINKFLAGS = $(OPTFLAGS) -static-libgcc -static-intel
 RENAME = rename_bin
 
 ##  Build:     Tinker
