--- configure.wrf.orig  2022-08-14 14:11:11.000000000 +0300
+++ configure.wrf       2022-08-14 14:54:35.000000000 +0300
@@ -119,7 +119,7 @@
                       $(WRF_SRC_ROOT_DIR)/frame/pack_utils.o

  LIB_EXTERNAL    = \
-                      -L$(WRF_SRC_ROOT_DIR)/external/io_netcdf -lwrfio_nf -L/opt/software/bundles/wrf/4-hdf5-netcdf-nvidia/lib -lnetcdff -lnetcdf     -L/opt/software/bundles/wrf/4-hdf5-netcdf-nvidia/lib -lhdf5hl_fortran -lhdf5_hl -lhdf5_fortran -lhdf5 -lm -lz
+                      -L$(WRF_SRC_ROOT_DIR)/external/io_netcdf -lwrfio_nf -L/opt/software/bundles/wrf/4-hdf5-netcdf-nvidia/lib -lnetcdff -lnetcdf     -L/opt/software/bundles/wrf/4-hdf5-netcdf-nvidia/lib -lhdf5_hl_fortran -lhdf5_hl -lhdf5_fortran -lhdf5 -lm -lz


 #### Architecture specific settings ####
@@ -136,17 +136,17 @@
 CCOMP           =       pgcc
 DM_FC           =       mpif90
 DM_CC           =       mpicc
-FC              =       time $(DM_FC)
+FC              =       $(DM_FC)
 CC              =       $(DM_CC) -DFSEEKO64_OK
 LD              =       $(FC)
 RWORDSIZE       =       $(NATIVE_RWORDSIZE)
 PROMOTION       =       -r$(RWORDSIZE) -i4
 ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM $(NETCDF4_IO_OPTS) -DRPC_TYPES=2
-CFLAGS_LOCAL    =       -w -O3 # -DRSL0_ONLY
+CFLAGS_LOCAL    =       -w -O3 -tp zen2 -Mcache_align # -DRSL0_ONLY
 LDFLAGS_LOCAL   =
 CPLUSPLUSLIB    =
 ESMF_LDFLAG     =       $(CPLUSPLUSLIB)
-FCOPTIM         =       -O3 #-fastsse -Mvect=noaltcode -Msmartalloc -Mprefetch=distance:8 -Mfprelaxed # -Minfo=all =Mneginfo=all
+FCOPTIM         =       -O3 -tp zen2 -Mcache_align #-fastsse -Mvect=noaltcode -Msmartalloc -Mprefetch=distance:8 -Mfprelaxed # -Minfo=all =Mneginfo=all
 FCREDUCEDOPT   =       $(FCOPTIM)
 FCNOOPT                =       -O0
 FCDEBUG         =       # -g $(FCNOOPT)  # -C -Ktrap=fp -traceback

