--- configure.wrf.orig  2023-04-08 22:30:28.000000000 +0300
+++ configure.wrf       2023-04-08 22:31:35.000000000 +0300
@@ -136,17 +136,17 @@
 CCOMP           =       pgcc
 DM_FC           =       mpif90
 DM_CC           =       mpicc
-FC              =       time $(DM_FC)
+FC              =       $(DM_FC)
 CC              =       $(DM_CC) -DFSEEKO64_OK
 LD              =       $(FC)
 RWORDSIZE       =       $(NATIVE_RWORDSIZE)
 PROMOTION       =       -r$(RWORDSIZE) -i4
 ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM $(NETCDF4_IO_OPTS)
-CFLAGS_LOCAL    =       -w -O3 -DRPC_TYPES=2 # -DRSL0_ONLY
+CFLAGS_LOCAL    =       -w -O3 -tp zen2 -Mcache_align -DRPC_TYPES=2
 LDFLAGS_LOCAL   =
 CPLUSPLUSLIB    =
 ESMF_LDFLAG     =       $(CPLUSPLUSLIB)
-FCOPTIM         =       -O3 #-fastsse -Mvect=noaltcode -Msmartalloc -Mprefetch=distance:8 -Mfprelaxed # -Minfo=all =Mneginfo=all
+FCOPTIM         =       -O3 -tp zen2 -Mcache_align
 FCREDUCEDOPT   =       $(FCOPTIM)
 FCNOOPT                =       -O0
 FCDEBUG         =       # -g $(FCNOOPT)  # -C -Ktrap=fp -traceback
